proto/cosmwasm/wasm/v1/ibc.proto

```diff
@@ -29,3 +29,9 @@ message MsgIBCSend {
 message MsgIBCCloseChannel {
   string channel = 2 [ (gogoproto.moretags) = "yaml:\"source_channel\"" ];
 }
+
+// MsgIBCSendResponse
+message MsgIBCSendResponse {
+  // Sequence number of the IBC packet sent
+  uint64 sequence = 1;
+}
```

Comment:
`proto/cosmwasm/wasm/v1/tx.proto` is not super consistent but usually we have the response type right after the message type. I think that's helpful for navigating. I'd move it above `message MsgIBCCloseChannel {`.